### PR TITLE
say: update page

### PR DESCRIPTION
**Pull request by chanibal**
A distinction that voice is language was needed when I translated to Polish, will be helpful here too.

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/osx/say.md [hunk 1]
```diff
@@ -15,10 +15,14 @@
 
 `say --voice={{voice}} --rate={{words_per_minute}} "{{I'm sorry Dave, I can't let you do that.}}"`
 
-- List the available voices:
+- List the available voices, different voices speak in different languages:
```
Comment (yutyo) [inline pages/osx/say.md:18]: Wouldn't `List the available voices (different voices speak in different languages):` be prettier and in accordance with other pages?

**Review by yutyo — APPROVED** [review]
LGTM

File: pages/osx/say.md (hunk 1 above)
Comment (CleanMachine1) [reply pages/osx/say.md:18]: Or 
```suggestion
- List the available voices with different voices which speak in different languages:
```

File: pages/osx/say.md [hunk 2]
```diff
@@ -15,10 +15,14 @@
...
 
 `say --voice="?"`
 
+- Say something in Polish:
+
+`say --voice={{Zosia}} "{{Litwo, ojczyzno moja!}}"`
```
Comment (CleanMachine1) [inline pages/osx/say.md:24]: Maybe `hello world` translated instead?

File: pages/osx/say.md (hunk 1 above)
Comment (chanibal) [reply pages/osx/say.md]: I think @yutyo's suggestion is better, updating.

**Review by kbdharun — APPROVED** [review]

**Review by CleanMachine1 — APPROVED** [review]

**Pull request merged**